core/templates/dev/head/pages/exploration_player/PlayerServices.js

```diff
@@ -260,8 +262,9 @@ oppia.factory('oppiaPlayerService', [
       isContentAudioTranslationAvailable: function(stateName) {
         return Object.keys(
           exploration.getAudioTranslations(stateName)).length > 0 ||
-          LanguageUtilService.supportsAutogeneratedAudio(
-            exploration.languageCode);
+          (LanguageUtilService.supportsAutogeneratedAudio(
```

Comment:
Should we find a way to reuse _isAutogeneratedAudioAllowed() (so you get the chrome check too)?

Reply:
good catch, I replaced it with a call to isAutogeneratedAudioAllowed()